vendor/tags.cncf.io/container-device-interface/pkg/cdi/cache_test_darwin.go

```diff
@@ -0,0 +1,26 @@
+//go:build darwin
+// +build darwin
+
+/*
+   Copyright © 2021 The CDI Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cdi
+
+import "syscall"
+
+func osSync() {
```

Comment:
@elezar This file should be called `cache_darwin_test.go`

Reply:
Noted. I will change this upstream.

Reply:
- opened https://github.com/cncf-tags/container-device-interface/pull/268

Reply:
Thanks. I haven't gotten around to it yet.

Reply:
No worries! I recalled this comment, and went looking if there was an updated tag. It was an easy change, so thought I'd just open a PR 😄